tests/modules/programs/lsd/default.nix

```
@@ -0,0 +1 @@
+{ example-settings = ./example-settings.nix; }
```

The test name should start with something that identifies the module.

```suggestion
{ lsd-example-settings = ./example-settings.nix; }
```